test/cmd-collect-delay.test.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path')
+const { test } = require('tap')
+const rimraf = require('rimraf')
+const ClinicFlame = require('../index.js')
+
+test('cmd - test collect - 1s collect delay', (t) => {
+  const tool = new ClinicFlame({ debug: true, collectDelay: 1000 })
+
+  function cleanup (err, dirname) {
+    t.ifError(err)
+    t.match(dirname, /^[0-9]+\.clinic-flame$/)
+
+    rimraf(dirname, (err) => {
+      t.ifError(err)
+      t.end()
+    })
+  }
+
+  const searchTree = (tree, target) => {
+    if (tree.name.includes(target)) {
+      return tree
+    }
+    for (const child of tree.children) {
+      const res = searchTree(child, target)
+
+      if (res) {
+        return res
+      }
+    }
+  }
+
+  tool.collect(
+    [process.execPath, path.join('fixtures', 'delay.js')],
```

Comment:
```suggestion
    [process.execPath, path.join('test', 'fixtures', 'delay.js')],
```
`npm test` runs the tests with the repo root as the working directory, so the path to `test/fixtures/delay.js` has to include `test/`. I think this is probably why the macOS tests are failing.